api/src/test/java/org/openmrs/test/BaseContextSensitiveTest.java

```diff
@@ -601,12 +601,29 @@ public void setAutoIncrementOnTablesWithNativeIfNotAssignedIdentityGenerator() t
 	}
 
 	/**
+<<<<<<< HEAD
+=======
+	 * Drops the not null constraint from the the specified column in the specified table
+	 *
+	 * @param columnName the column from which to remove the constraint
+	 * @param tableName the table that contains the column
+	 * @throws SQLException
+	 */
+	protected void dropNotNullConstraint(String tableName, String columnName) throws SQLException {
+		if (!useInMemoryDatabase()) {
+			throw new RuntimeException("Altering column nullability is not supported for a non in-memory database");
+		}
+		final String sql = "ALTER TABLE " + tableName + " ALTER COLUMN " + columnName + " SET NULL";
+		DatabaseUtil.executeSQL(getConnection(), sql, false);
+	}
+	/**
+>>>>>>> 743de0d4d... TRUNK-5980: BaseContextSensitiveTest to allow further protected overrides. (#3664)
```

Comment:
oops And here as well .

Reply:
Thanks @tendomart  Have resolved the conflicts it LGTM  cc @dkayiwa  @mks-d